README.md

```diff
@@ -54,44 +55,45 @@ To install this to your Raspberry Pi, you need the following things:
   + After the writing process is finished, insert the SD card with the new OS version into your Raspberry Pi and turn it on.
   + Done.
 
-
 ### Set up environment
-Clone or download this repo
-
-Navigate with the terminal to where it is
-
-Create a virtual environment
-
-`python3 -m venv PVM`
 
-Open your venv
+```bash
+# Clone the repo
+git clone https://github.com/omarcostahamido/PVM.git
 
-`source PVM/bin/activate`
+cd PVM
 
-(optional) update pip and setuptools
+# Create a virtual environment
+python3 -m venv PVM
 
-`pip install --upgrade pip setuptools`
```

Comment:
simply delete this?
(Actually, I wonder if there is a way to simply upgrade these from within the `requirements.txt`)
In my experience I've noted that not updating pip and setuptools on a venv sometimes leads to problems when installing packages...

Reply:
Readded

Reply:
🙌